enterprise_access/apps/provisioning/admin.py

```diff
@@ -364,6 +364,7 @@ def _process_form(self, form, request):
                 catalog_step_input,
                 agreement_step_input,
                 plan_step_input,
+                {},  # TODO: pass a real first paid dict argument
```

Comment:
hm, interesting that this was passing after my changes to add three more workflow steps. 🤷